hawkular-command-gateway-war/pom.xml

```diff
@@ -72,11 +72,6 @@
     </dependency>
 
     <dependency>
-      <groupId>org.hawkular.inventory</groupId>
-      <artifactId>hawkular-inventory-json-helper</artifactId>
-    </dependency>
```

Comment:
OK, I am accepting that clients will only know the resource identified by `resourcePath` and that they will not know by which feed it is monitored.

This will require from the gateway to be able to analyze the internal structure of `resourcePath`s in order to (typically) extract the feedId to send the message to. I can accept this as a part of the general task of routing.

> This is why I liked the idea of the command implementations on the server being the ones to determine where to forward the messages. Because some commands a client will tell us a single resourcePath (which are then parsed via the inventory API to determine what feed is managing that resource), others may provide a group path (which could then be used to query the inventory to determine which resources are currently in that group, and then from that which feed(s) manage those resources keeping in mind one feed many manage more than one of these resources), others may be an array of resourcePaths.

Yes, you named a couple of (well, exactly three) ways how to compute the routing info out of a UI request. I accept that these and similar ways of computing must be doable with our API. But I strongly vote for keeping these routing computing code as concentrated as possible, much to the opposite of the present practice where every request class has a special gateway command and a special bus listener (all of them doing the same thing). To cover the commands (except for Echo) we have now, only one routing info provider should suffice - namely the one that takes `resourcePath` and extracts `feedId` out of it.